_config/logging.yml

```diff
@@ -38,6 +38,7 @@ SilverStripe\Core\Injector\Injector:
       - "error"
     properties:
       Formatter: %$FriendlyErrorFormatter
```

Comment:
Or given that the interface is `Monolog\Formatter\FormatterInterface`, we could use dot notation to declare the named variant of the interface provided.

`Monolog\Formatter\FormatterInterface.friendly`

and

`Monolog\Formatter\FormatterInterface.cli`

Depending on how pure you want your treatment of interfaces to be. :)